scripts/zones/West_Sarutabaruta_[S]/mobs/Toad.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------
+-- Area: West Saurtabaruta (S)
+--  Mob: Toad
+-- Note: Place holder Ramponneau
+-----------------------------------
+
+require("scripts/zones/West_Sarutabaruta_[S]/MobIDs");
+
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob,killer)
+
+    local mob = mob:getID();
+    if mob == Ramponneau_PH then

```

Comment:
Also:
`if (stuff) then`
 not
 `if stuff then`
for styling consistencies sake